testing/src/main/resources/certs/README

```diff
@@ -59,11 +59,25 @@ $ rm server1.key.rsa
 $ openssl req -new -key server1.key -out server1.csr -config server1-openssl.cnf
 
 When prompted for certificate information, everything is default except the
-common name which is set to *.test.google.com.
+common name, which is set to *.test.google.com.
 
 $ openssl ca -in server1.csr -out server1.pem -keyfile ca.key -cert ca.pem -verbose -config server1-openssl.cnf -days 3650 -extensions v3_req -updatedb
 $ openssl x509 -in server1.pem -out server1.pem -outform PEM
 
+localhost_server is issued by CA with the common name "localhost":
```

Comment:
I don't believe these additional certs are actually necessary. The client/server doesn't need to trust its own certificate, so the server could use server1.pem without overriding the trust manager (as we do [elsewhere](https://github.com/grpc/grpc-java/blob/master/interop-testing/src/main/java/io/grpc/testing/integration/TestServiceServer.java#L138)). We would just choose to override the trust manager when we wanted the client certificate to be valid.